CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.1.1](https://github.com/sonata-project/SonataPageBundle/compare/3.1.0...3.1.1) - 2016-08-01
```

Comment:
https://github.com/sonata-project/SonataAdminBundle/pull/4102
Last time he told me to fill the changelog, or I missunderstand the request